lib/jekyll.rb

```diff
@@ -154,29 +155,29 @@ def sites
     def sanitized_path(base_directory, questionable_path)
       return base_directory if base_directory.eql?(questionable_path)
 
-      questionable_path.insert(0, '/') if questionable_path.start_with?('~')
+      questionable_path.insert(0, "/") if questionable_path.start_with?("~")
       clean_path = File.expand_path(questionable_path, "/")
-      clean_path.sub!(/\A\w\:\//, '/')
+      clean_path.sub!(%r!/\A\w\:\//!, "/")
```

Comment:
The `//` at the beginning and end didn't get removed, this will break the current API.